OlinkAnalyze/R/olink_normalization.R

```diff
@@ -180,6 +205,25 @@
 #'   overlapping_samples_df1 = df1_subset,
 #'   reference_medians = ref_med_df
 #' )
+#'
+#' # cross-product normalization
+#'
+#' # get reference samples
+#' overlap_samples_product <- intersect(
+#'   x = unique(OlinkAnalyze:::data_ht$SampleID),
+#'   y = unique(OlinkAnalyze:::data_3k$SampleID)
+#' ) |>
+#'   (\(.) .[!grepl("CONTROL", .)])()
+#'
+#' # normalize
+#' olink_normalization(
+#'   df1 = OlinkAnalyze:::data_ht,
```

Comment:
after merging PR #454, do we need to change instances like OlinkAnalyze:::data_ht to OlinkAnalyze:::data_ht_small?